### PR TITLE
Update 1-js/11-async/04-promise-error-handling

### PR DESCRIPTION
**Pull request by lycheeEng**
Based on English version: [49381c3551df21a1f2551b665bd46a0fb124f8fd](https://github.com/javascript-tutorial/en.javascript.info/commit/49381c3551df21a1f2551b665bd46a0fb124f8fd)

### CODE REVIEW

**Comment by MartinsYong** [comment]
校对认领~

File: 1-js/11-async/04-promise-error-handling/article.md [hunk 1]
```diff
@@ -74,48 +74,48 @@ new Promise((resolve, reject) => {
...
+## 重新抛出（Rethrowing）
 
-As we already noticed, `.catch` behaves like `try..catch`. We may have as many `.then` handlers as we want, and then use a single `.catch` at the end to handle errors in all of them.
+正如我们已经注意到的， `.catch` 表现得像 `try..catch`。我们可能有许多的 `.then` 处理程序，然后在最末端使用一个 `.catch` 处理上面的所有错误。
 
-In a regular `try..catch` we can analyze the error and maybe rethrow it if can't handle. The same thing is possible for promises.
+在常规 `try..catch` 中，我们可以分析错误，当我们无法处理的时候可能还会重新抛出（rethrow）它。对于 promise 来说也可以这样做。
 
-If we `throw` inside `.catch`, then the control goes to the next closest error handler. And if we handle the error and finish normally, then it continues to the closest successful `.then` handler.
+如果我们在 `.catch` 内 `throw`，此时控制移交到下一个最近的处理程序。如果我们处理错误并正常完成，那么它将继续到最近的成功的 `.then` 处理程序。
```
Comment (MartinsYong) [inline 1-js/11-async/04-promise-error-handling/article.md]: [下一个最近的处理程序] -> [下一个最近的错误处理程序]

File: 1-js/11-async/04-promise-error-handling/article.md [hunk 2]
```diff
@@ -127,12 +127,12 @@ new Promise((resolve, reject) => {
 }).then(() => alert("Next successful handler runs"));
 ```
 
-Here the `.catch` block finishes normally. So the next successful `.then` handler is called.
+这里 `.catch` 块正常完成。因此就会调用下一个成功的 `.then` 处理程序。
 
-In the example below we see the other situation with `.catch`. The handler `(*)` catches the error and just can't handle it (e.g. it only knows how to handle `URIError`), so it throws it again:
+在下面的例子中，我们可以看到 `.catch` 的另一种情况。`(*)` 行的处理程序捕获错误但无法处理它（例如，它只知道如何处理 `URIError` 错误），所以它将错误中心抛出：
```
Comment (MartinsYong) [inline 1-js/11-async/04-promise-error-handling/article.md]: [所以它将错误中心抛出] -> [所有它再次被抛出]

Comment (lycheeEng) [reply 1-js/11-async/04-promise-error-handling/article.md]: `所有它再次被抛出` 这里应该是 `所以` 吧？

**Comment by lycheeEng** [comment]
修改完成，感谢校对~

File: 1-js/11-async/04-promise-error-handling/article.md (hunk 2 above)
Comment (MartinsYong) [reply 1-js/11-async/04-promise-error-handling/article.md]: 嗯嗯

**Pull request merged**